setup.py

```diff
@@ -0,0 +1,30 @@
+#!/usr/bin/env python3
+from setuptools import setup
+import os
+
+
+# README as long_description
+current_directory = os.path.dirname(os.path.abspath(__file__))
+try:
+    with open(os.path.join(current_directory, "README.md"), encoding="utf-8") as f:
+        long_description = f.read()
+except Exception:
+    long_description = ""
+
+setup(
+    name="zoomdl",
+    version="2023.01.04_homebrew",
+    description="Download Zoom recorded meetings easily",
+    url="https://github.com/Battleman/zoomdl",
+    author="Olivier Cloux",
+    license="MIT",
```

Comment:
The license is GPL-V3

Reply:
I must have missed that in the template that I used; fixed it